cmd/derper/derper.go

```diff
@@ -39,6 +38,7 @@ var (
 	dev           = flag.Bool("dev", false, "run in localhost development mode")
 	addr          = flag.String("a", ":443", "server address")
 	configPath    = flag.String("c", "", "config file path")
+	certMode      = flag.String("certmode", "letsencrypt", "mode for getting a cert. possible options: manual, letsencrypt")
```

Comment:
did you consider making this just be a `--letsencrypt=true` bool flag? Do you foresee other cert modes in the future?

Reply:
Yes. Letsencrypt is just an ACME provider and maybe someone want to use alternate provider (like [DigiCert](https://docs.digicert.com/certificate-tools/Certificate-lifecycle-automation-index/automation-user-guide/install-automation-agent-webserver/)). Maybe one day, some one want to implement a "acmev2" mode, and allowing customize endpoint, but that's out of the scope of this PR.